VERSION

```diff
@@ -1 +1 @@
-7.2.0
\ No newline at end of file
+7.2.0a1
```

Comment:
I am not sure if this versioning will work with the version checking in  `utils/version`

Reply:
all tests passed and I added some logic in the bittensor.__init__.py for __version__